packages/client/components/TransactionDetails.tsx

```diff
@@ -0,0 +1,255 @@
+                                <Text color="success.600">Confirmed</Text>
+                            </Badge>
+                        </HStack>
+                    </Box>
+                    <Box style={styles.subItemWrapper}>
+                        <Text color="text.500" style={styles.elementInformationTitle}>
+                            From
+                        </Text>
+                        <HStack space="10px">
+                            <Text style={styles.elementInformationText}>{transaction.walletIn}</Text>
+                            {isIncommingTransaction ? (
+                                <Pressable onPress={() => copyToClipboard(transaction.walletIn)}>
+                                    <FontAwesomeIcon icon={faCopyCustom} style={styles.copyIcon} size={20} />
+                                </Pressable>
+                            ) : (
+                                <></>
+                            )}
+                        </HStack>
+                    </Box>
+                    <Box style={styles.subItemWrapper}>
+                        <Text color="text.500" style={styles.elementInformationTitle}>
+                            To
+                        </Text>
+                        <HStack space="10px">
+                            <Text style={styles.elementInformationText}>{transaction.walletOut}</Text>
+                            {isIncommingTransaction ? (
+                                <></>
+                            ) : (
+                                <Pressable onPress={() => copyToClipboard(transaction.walletOut)}>
+                                    <FontAwesomeIcon icon={faCopyCustom} style={styles.copyIcon} size={20} />
+                                </Pressable>
+                            )}
+                        </HStack>
+                    </Box>
+                </VStack>
+            </Box>
+        );
+    }
+    function renderTransactionFee() {
+        // TODO
+        // Get Gas
+        // Get Gas Limit
+        return (
+            <Box style={styles.itemWrapper}>
+                <VStack>
+                    <Box>
+                        <Text style={styles.categoryTitle}>Transaction Fee</Text>
+                    </Box>
+                    <Box style={styles.subItemWrapper}>
+                        <Text color="text.500" style={styles.elementInformationTitle}>
+                            Gas Price
+                        </Text>
+                        <Text>0.0000000000012323 ETH</Text>
+                    </Box>
+                    <Box style={styles.subItemWrapper}>
+                        <HStack>
+                            <Text>Gas Limit</Text>
+                            <Text color="text.500" style={styles.gasLimit}>
+                                100 000
+                            </Text>
+                        </HStack>
+                    </Box>
+                </VStack>
+            </Box>
```

Comment:
VStack implements box for example so you can just move the styling to the sub component and remove half of this nesting
```suggestion
        <VStack style={styles.itemWrapper}>
            <Text style={styles.categoryTitle}>Transaction Fee</Text>
            <Box style={styles.subItemWrapper}>
                <Text color="text.500" style={styles.elementInformationTitle}>
                    Gas Price
                </Text>
                <Text>0.0000000000012323 ETH</Text>
            </Box>
            <HStack  style={styles.subItemWrapper}>
                <Text>Gas Limit</Text>
                <Text color="text.500" style={styles.gasLimit}>
                    100 000
                </Text>
            </HStack>
        </VStack>
```